mpitests/test_native_mcm.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for default qubit preprocessing."""
+import numpy as np
+import pennylane as qml
+import pytest
+from conftest import LightningDevice, device_name
+from mpi4py import MPI
+
+if not LightningDevice._CPP_BINARY_AVAILABLE:  # pylint: disable=protected-access
+    pytest.skip("No binary module found. Skipping.", allow_module_level=True)
+
+
+def test_unspported_mid_measurement():
+    """Test unsupported mid_measurement for lightning.gpu-mpi."""
```

Comment:
Not related to the support: Could you ensure that we're referencing the device with the same consistent style everywhere? `Lightning-GPU-MPI` could be a good candidate.
```
    """Test unsupported mid_measurement for Lightning-GPU-MPI."""
```